### PR TITLE
[zh] sync /news/releases/1.15.x/announcing-1.15/_index.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/news/releases/1.15.x/announcing-1.15/_index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+您还可以加入 [Discuss Istio](https://discuss.istio.io/)的对话，或加入我们的 [Slack 工作区](https://slack.istio.io/)。
+您想直接为 Istio 做出贡献吗？您可以查找并加入我们其中任意一个[工作组](https://github.com/istio/community/blob/master/WORKING-GROUPS.md)以帮助我们改进 Istio。
+
+## KubeCon NA 上的 Istio{#istio-at-kubecon-na}
+
+Istio 将于今年 10 月在底特律参加 KubeCon 北美会议，请不要错过 TOC 成员 John Howard 与来自 Microsoft 的 Keith Mattix 的演讲，您将在其中了解通用服务网格 API 的新 GAMMA 计划。还有关于在生产环境中动态测试版本和分片应用的分散路由的讨论。而且，如果这还不够的话，还有一个专门用于服务网格的整个同地活动——ServiceMeshCon NA。与项目主席 Craig Box（来自 Google）和 Lin Sun（来自 Solo.io）一起讨论服务网格技术的发展历程。
+
+## CNCF 进度更新{#cncf-progress-update}
+
+4 月，我们宣布 Istio 已被提议成为 CNCF 的孵化项目。我们的团队一直在努力准备我们的申请，TOC 目前处于公开征求意见阶段。如果您想参加，请看这个帖子！
```
Comment (rootsongjc) [inline content/zh/news/releases/1.15.x/announcing-1.15/_index.md:52]: 「请看这个帖子」这里是不是应该有个链接？不过我发现英文版的原文里也没有链接，这里可能漏掉了？

File: content/zh/news/releases/1.15.x/announcing-1.15/_index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+aliases:
+- /zh/news/announcing-1.15
+- /zh/news/announcing-1.15.0
+---
+
+我们很高兴地宣布发布 Istio 1.15！
+
+{{< relnote >}}
+
+这是 2022 年的第三个 Istio 版本。我们要感谢整个 Istio 社区帮助发布 Istio 1.15.0。
```
Comment (litong01) [inline content/zh/news/releases/1.15.x/announcing-1.15/_index.md:18]: probably change it to something like this?

这是 2022 年发布的第三个 Istio 版本.

File: content/zh/news/releases/1.15.x/announcing-1.15/_index.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+特别感谢来自 Google 的发布经理 Sam Naser 和 Aryan Gupta、来自 Intel 的 Ziyang Xiao 和来自 Solo.io 的 Daniel Hawton。
```
Comment (litong01) [inline content/zh/news/releases/1.15.x/announcing-1.15/_index.md:19]: can we add the following?
我们特别感谢来自

**Review by litong01 — CHANGES_REQUESTED** [review]

File: content/zh/news/releases/1.15.x/announcing-1.15/_index.md (hunk 1 above)
Comment (yanrongshi) [reply content/zh/news/releases/1.15.x/announcing-1.15/_index.md:52]: 原文中没有这个链接，我去官网上查了，也没有这个类型的帖子

Comment (rootsongjc) [reply content/zh/news/releases/1.15.x/announcing-1.15/_index.md:52]: 如果没有的话不如先把这句话删掉好了，省的让大家困惑。

**Review by litong01 — APPROVED** [review]
Thanks for your contributions.

**Review by Xunzhuo — APPROVED** [review]

**Pull request merged**